keyboards/crkbd/keymaps/kidbrazil/rules.mk

```diff
@@ -9,9 +9,4 @@ OLED_DRIVER_ENABLE = yes
 
 # If you want to change the display of OLED, you need to change here
 SRC +=  ./keyboards/crkbd/keymaps/kidbrazil/logo_reader.c \
-        #./lib/rgb_state_reader.c \
-        #./lib/logo_reader.c \
-        #./lib/keylogger.c \
-        # ./lib/mode_icon_reader.c \
-        # ./lib/host_led_state_reader.c \
-        # ./lib/timelogger.c \
+				./keyboards/crkbd/keymaps/kidbrazil/layer.c
```

Comment:
Just a heads up, you don't need the fill path for either of these files.  You should be okay with:
```make
SRC += logo_reader.c \
              layer.c
```

This is because the keymap path is added to the "virtual path" for compiling, so you don't need to use the full path.

Reply:
Awesome thanks @drashna I will correct that later today and get it pushed in.